### DISCUSSION TITLE
[Feature] untag person without tagging as new person

### DISCUSSION THREAD

**Discussion by igor47** [immich-app/immich#5962] [Feature Request]
### The feature

Suppose I have a mis-identified person in the photo. Currently, when I click on `Edit People` from the info sidebar, I get a little blue `-` button on every face. When I click it, I am prompted to pick an existing person, or create a new person. I actually just want to untag the person.

Maybe there could be two buttons -- the `-` sign to just remove the tag, and a pencil icon to edit the person?

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by hirenshah** [comment]
Agreed.  I have a photo with these two additional people tagged and can't seem to find a way to remove the tag.

![image](https://github.com/immich-app/immich/assets/418112/2dc54c3b-db10-4eec-92bb-b1e98418d69a)

**Reply by tbleiker** [reply]
> Agreed.  I have a photo with these two additional people tagged and can't seem to find a way to remove the tag.
> 
> ![image](https://github.com/immich-app/immich/assets/418112/2dc54c3b-db10-4eec-92bb-b1e98418d69a)
> 

With the new face grouping (DBSCAN clustering) in v1.93, people which are only found once or twice should not be shown anymore. You can adjust this limit in the settings: Machine Learning Settings -> Facial Recognition -> Min Recognized Faces

**Reply by hirenshah** [reply]
I've set that value to 5, but it doesn't help when this is somehow tagged in 11 completely different and independant photos:
![image](https://github.com/immich-app/immich/assets/418112/76b8c7d9-1eee-4644-bb6a-5b24585aa840)

**Comment by tbleiker** [comment]
I would like to untag/hide "faces" which are wrongly detected as faces.

(I slightly lowered "min detection score" to get more faces detected. Now I get some false positives...)

**Comment by aisbergde** [comment]
this is what I would like to do as part of #6529 - untag for me means: remove the recognition tag and move the face into the set of detected but not recognized faces.

**Comment by bo0tzz** [comment]
#5276

**Comment by MacBadger** [comment]
Yes we really need this. I take a lot of photos with other faces in - crowds at events, record sleeves in the background with faces on. I get these 'tagged' as people I know and do group. So there are already lots of photos of the person they think it is. Just need a way to say "not a face" or at least "not a face I care about" This is the only major issue I have.

**Comment by jongalli** [comment]
The only workaround I found round this was to:

- select "pencil icon" next to face that has been incorrectly tagged (or you want removed)
- select "+" icon to create a new person
- select done
- select done again

the face is now no longer tagged to a person of your interest
